### DISCUSSION TITLE
Using remark/md files with image from public folder

### DISCUSSION THREAD

**Discussion by tettoffensive** [vercel/next.js#40599] [Help] [answered]
I'm starting with the [blog-starter](https://github.com/vercel/next.js/tree/canary/examples/blog-starter) example. Which roughly follows this article here: https://nextjs.org/blog/markdown.

I have markdown files which could have many different images. Most of the examples I've seen have image urls, starting with "https://", but in my case **I just want to show images that are in my public folder. Is there a good way to do this?** Like maybe a remark plugin?

```md
---
title: 'title'
---

Here is some content

![Example image](/assets/image.jpeg)

![Example image](/assets/another-image.jpeg)
```

```ts
export default async function markdownToHtml(markdown: string) {
  const result = await remark().use(html).process(markdown)
  return result.toString()
}
```

**Comment by icyJoseph** [comment, marked as answer]
> I just want to show images that are in my public folder. Is there a good way to do this? 

That starter repo you linked uses images from the public folder, for the front matter anyway, but you can also add this:

```mdx
This is an image:

![Example image](/assets/blog/preview/cover.jpg)
```

I added that to the [preview page](https://github.com/vercel/next.js/blob/canary/examples/blog-starter/_posts/preview.md), and it worked, but in dev and production mode.

Have I misunderstood? What are you exactly after? Or perhaps this was what you needed?

**Reply by tettoffensive** [reply]
Yes for some reason it didn't work at first. And then it started working after messing with it. Not sure what the issue was.
